tests/edge/test_edge.py

```diff
@@ -290,6 +290,7 @@ def mocked_load_data(datadir, pairs=[], timeframe='0m', refresh_pairs=False,
 def test_edge_process_downloaded_data(mocker, edge_conf):
     freqtrade = get_patched_freqtradebot(mocker, edge_conf)
     mocker.patch('freqtrade.exchange.Exchange.get_fee', MagicMock(return_value=0.001))
+    mocker.patch('freqtrade.data.history.refresh_data', MagicMock())
```

Comment:
pls verify it, am I right with adding this into these tests?

Reply:
i guess so - yes - tests pass which means there's no typo in the object resolution (these are sometimes difficult to spot ...)

and not calling "refresh_data" from here is how it used to work before, too, by mocking load_data.